recipes/hafez/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hafez" %}
+{% set ucname = "hafeZ" %}
```

Comment:
Just use name, but with the capital letter. Isn't that working?

Reply:
Sorry, I'd copied the ucname part from an existing recipe but I've updated it now to use only name.